### ISSUE TITLE
Is this package maintained?

### ISSUE THREAD

**Issue by GShotwell** [looker-open-source/lookr#38]
We're thinking of purchasing Looker and as part of that we're looking into how well Looker integrates to our existing R and Python workflows. I see that this package hasn't had any activity in 18 months, and also seems to be one API version behind the Looker API. I also see a number of issues which seem to be related to installing the package on windows and using recent versions of R which haven't been updated. 

Can someone chime in on whether this package is still supported? If not, it might be good to add that to the README.

**Comment by aaron-horowitz** [comment]
If it helps at all for the folks at Looker considering this, we've abandoned both the R and Python APIs entirely and basically use direct requests through http requests. I have no idea if this is the "right" answer, but found the original APIs sufficiently incomplete that this was the right choice for us.

**Comment by jkaster** [comment]
Thanks for bringing this up! We've just been discussing the update to this internally.

This SDK has been caught in the transition to our new [SDK code generator](https://github.com/looker-open-source/sdk-codegen). 

Work on a new version of `LookR` will begin after we have the Swift and Kotlin SDKs produced by our code generator ready for beta, but there's no specific date for that yet.

The first step for any of our new SDKs is developing the run-time library (RTL), as described in https://github.com/looker-open-source/sdk-codegen/blob/master/rationale.md#hand-written-run-time-libraries. That part of the SDKs is almost completely generic (IOW, not specific to Looker). If you have any interest in participating in the development of the `R` RTL, please let me know. Once the new `R` RTL exists, it should be a month or less of development work to generate the methods and models for `R`.

**Disclaimer**: I'm not an `R` language expert, so these are just guesses and hand-wavy right now. I'm one of the authors of the SDK Codegen project, so I know pretty well what's involved in producing the new `LookR`, but not specifically what the `R` support will look like.

**Side note**: You can get our new, supported Python SDK produced by that code generator from https://pypi.org/project/looker-sdk/. (We never had an officially supported Python SDK before that was released.)

Let me know if you have any other questions.

**Comment by dthorpe** [comment]
I don't believe any new work is planned for this R package. It was created and is maintained as a volunteer effort, not an officially supported Looker product.

We intend to support R as an output of our Looker Client SDK project, which is in active development: https://github.com/looker-open-source/sdk-codegen

One stumbling block for that intent is finding an R language expert to advise / help us produce an R SDK and validate any R client SDK we produce. Volunteers welcome!  :)

**Comment by dthorpe** [comment]
erp.  Jinx!  :)

**Comment by GShotwell** [comment]
Ah great to hear that things are moving forward. I'd suggest adding some text to the README or a [lifecycle badge](https://www.tidyverse.org/lifecycle/) to express that it's not  under active development. 

I'd be happy to help with the R binding, I've worked on a few API wrappers previously, though none that relied on a codegen process. Let me know what the best process is to get started.

**Comment by jkaster** [comment]
Good suggestion on documenting the status of LookR. We'll get to that hopefully next week!

I'm really happy to hear about the R binding. I'm meeting with an R aficionado here at Looker soon, and I'll have the start of a plan then. Just having someone who knows R who can say "Yuck, that's an ugly pattern" (or not) will be really valuable. If you read the rationale document linked above you should have a good idea of what we're planning.

Hopefully there will be a document on how to create an SDK for your chosen language available in the next couple weeks, if I can carve out some time for it.

**Comment by scrane-amplify** [comment]
+1 for documenting the status of this repo. I see that update was never made. I'd also be happy to contribute to an R SDK however might be helpful.

**Comment by jkaster** [comment]
Work will commence on the update to LookR shortly. Please stay tuned.

**Comment by rsaporta** [comment]
Howdie @jkaster, @dthorpe,  curious if any update on this?   thanks!

**Comment by jkaster** [comment]
Thanks for asking! Not yet, although we did just land a branch for Look#,
the C# SDK in the SDK codegen project. LookR is next on our list to look at
again.
